### PR TITLE
ARC-2195: introduce new transcribe parameters

### PR DESCRIPTION
**Pull request by ademgcl**

### CODE REVIEW

**Comment by ademgcl** [comment]
introduces new parameters for transcribe pathway:
- highlightWords
- maxLineWidth
- maxLineCount
- maxWordsPerLine

File: server/plugins/openAiWhisperPlugin.js [hunk 1]
```diff
@@ -39,6 +39,12 @@ function alignSubtitles(subtitles, format) {
         const subtitle = subtitles[i];
         result.push(...shiftSubtitles(subtitle, i * offset));
     }
+    
+    //if content has needed html style tags, keep them
+    for(const obj of result) {
```
Comment (jmacAJ) [inline server/plugins/openAiWhisperPlugin.js:44]: alignSubtitles could use some error handling.  You're having a lot of faith here that result is in the right form.

**Review by jmacAJ — CHANGES_REQUESTED** [review]
Need to add a little error handling to the plugin to make it more robust.

File: server/plugins/openAiWhisperPlugin.js (hunk 1 above)
Comment (ademgcl) [reply server/plugins/openAiWhisperPlugin.js]: added try-catch and obj field checking

**Review by jmacAJ — APPROVED** [review]

**Pull request merged**